### PR TITLE
Update index.html.src

### PR DESCRIPTION
**Pull request by KennethL**
Update the entry page for the documentation. Will come more before OTP 26

### CODE REVIEW

File: system/doc/top/templates/index.html.src [hunk 1]
```diff
@@ -200,14 +199,12 @@ module and function name completion (tab) if the module is loaded.
 <p>
 
 <li>OpenSource users can ask questions
-and share experiences on the <a href="http://www.erlang.org/static/doc/mailinglist.html">
-Erlang questions mailing list</a>.
+and share experiences on <a href="http://erlangforums.com">
```
Comment (max-au) [inline system/doc/top/templates/index.html.src:202]: Could you change it to `https` ?

Here, and pretty much everywhere else.

Comment (KennethL) [reply system/doc/top/templates/index.html.src:202]: will do that and more changes before OTP 26, thanks for the suggestion

Comment (KennethL) [reply system/doc/top/templates/index.html.src:202]: Closes #5048 as well

**Comment by KennethL** [comment]
continue work in a new PR

**Pull request closed without merging**